pyproject.toml

```
@@ -11,11 +11,43 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "velocity_modelling"
-authors = [{ name = "QuakeCoRE" }]
+authors = [{ name = "ucgmsim" }]
 description = "Velocity modelling library"
 readme = "README.md"
 requires-python = ">=3.11"
```

This needs to be changed because numba doesn't support 3.14. 


```suggestion
requires-python = ">=3.11,<3.14"
```